src/app/components/header/header.component.html

```diff
@@ -25,7 +25,7 @@
     </button>
   </mat-menu>
 
-  <button mat-button [matMenuTriggerFor]="championshipsMenu">
+  <button *ngIf="championshipsService.items.length > 1" mat-raised-button color="primary" [matMenuTriggerFor]="championshipsMenu">
     <span>{{ championshipsService.selected }}</span>
```

Comment:
(opt) what do you think about text inside a button instead to put a span?

Reply:
Makes sense, that span is redundant